build.gradle

```diff
@@ -17,16 +17,37 @@ subprojects {
     }
     apply plugin: 'java'
     apply plugin: 'com.diffplug.gradle.spotless'
-//    apply plugin: 'signing'
 
     spotless {
         java {
             indentWithSpaces(4)
             removeUnusedImports()
         }
     }
+}
+
+configure([project(':hydra-client'), project(':hydra-common'), project('hydra-gradle-plugin')]) {
+    task sourceJar(type: Jar) {
+        classifier = 'sources'
+        from sourceSets.main.allJava
+    }
+    task javadocJar(type: Jar, dependsOn: javadoc) {
+        classifier = 'javadoc'
+        from javadoc
+    }
+}
+
+configure (project(':hydra-server')) {
+    apply plugin: 'org.springframework.boot'
+    apply plugin: 'io.spring.dependency-management'
```

Comment:
In case you're unaware, this uses maven-style rules for dependency resolution, so versions of dependencies explicitly declared in a `dependencies` block will override transitive dependencies, similar to declaring every dependency with `force = true`. This may or may not be the behavior you want.

Reply:
Yeah, in this case it is fine.